rust/item/src/lib.rs

```diff
@@ -19,13 +19,14 @@ use anchor_lang::{
     },
     AnchorDeserialize, AnchorSerialize, Discriminator,
 };
+anchor_lang::declare_id!("CKAcdJsyzBxHJRHgKVEsVzjX9SNcvut8t3PUD34g7ry4");
 use anchor_spl::token::{Mint, Token, TokenAccount};
 use arrayref::array_ref;
 use std::str::FromStr;
-anchor_lang::declare_id!("itemX1XWs9dK8T2Zca4vEEPfCAhRc7yvYFntPjTTVx6");
 pub const PREFIX: &str = "item";
 pub const MARKER: &str = "marker";
 pub const PLAYER_ID: &str = "p1exdMJcjVao65QdewkaZRUnU6VPSXhus9n2GzWfh98";
+pub const NAMESPACE_ID: &str = "AguQatwNFEaZSFUHsTj5fcU3LdsNFQLrYSHQjZ4erC8X";
 pub const STAKING_ID: &str = "stk9HFnKhZN2PZjnn5C4wTzmeiAEgsDkbqnHkNjX1Z4";
 pub const RENT_ID: &str = "SysvarRent111111111111111111111111111111111";
```

Comment:
Can we keep the old program IDs?

Reply:
Once I get the keypairs I will revert this change, its only temporary and I wont merge the PR until I do

Reply:
I dont think the anchor provider is what I need to use right? https://github.com/raindrops-protocol/raindrops/blob/4f1fd9c8847ae2f6da5a45a2ea6a743d084b2195/js/lib/src/state/common.ts#L50